### ISSUE TITLE
plugin breaks on docusaurus canary from refactored theme translation api

### ISSUE THREAD

**Issue by duanwilliam** [cmfcmf/docusaurus-search-local#82]
### Bug description

https://github.com/facebook/docusaurus/pull/5849 refactors the theme translation api and moves 
 the `readDefaultCodeTranslationMessages` function, used in this plugin, from `@docusaurus/utils` into a separate package, `@docusaurus/theme-translations`. This should probably be updated when Docusaurus get its next stable release in `2.0.0-beta.10`.

### Reproduction

try `yarn start` on canary build (not sure which one corresponds with the breaking pr but tested on 0.0.0-4248).

#### Package Versions

```
├─ @cmfcmf/docusaurus-search-local@0.8.0
├─ @docusaurus/core@0.0.0-4248
├─ @docusaurus/cssnano-preset@0.0.0-4248
├─ @docusaurus/lqip-loader@0.0.0-4248
├─ @docusaurus/mdx-loader@0.0.0-4248
├─ @docusaurus/module-type-aliases@0.0.0-4248
├─ @docusaurus/plugin-content-blog@0.0.0-4248
├─ @docusaurus/plugin-content-docs@0.0.0-4248
├─ @docusaurus/plugin-content-pages@0.0.0-4248
├─ @docusaurus/plugin-debug@0.0.0-4248
├─ @docusaurus/plugin-google-analytics@0.0.0-4248
├─ @docusaurus/plugin-google-gtag@0.0.0-4248
├─ @docusaurus/plugin-ideal-image@0.0.0-4248
├─ @docusaurus/plugin-sitemap@0.0.0-4248
├─ @docusaurus/preset-classic@0.0.0-4248
├─ @docusaurus/react-loadable@5.5.2
├─ @docusaurus/responsive-loader@1.5.0
├─ @docusaurus/theme-classic@0.0.0-4248
├─ @docusaurus/theme-common@0.0.0-4248
├─ @docusaurus/theme-search-algolia@0.0.0-4248
├─ @docusaurus/theme-translations@0.0.0-4248
├─ @docusaurus/utils-common@0.0.0-4248
├─ @docusaurus/utils-validation@0.0.0-4248
└─ @docusaurus/utils@0.0.0-4248
```

#### Node Version

`v14.15.0`

#### Docusaurus Environment

- using `preset-classic`
- plugin config: `[require.resolve('@cmfcmf/docusaurus-search-local'), {}]`

**Comment by Josh-Cena** [comment]
@cmfcmf I'm encountering the same issue. In fact I'm encountering _lots_ of pain points while using this plugin. As a Docusaurus maintainer myself, may I ask you to kindly give me write permissions so I can help modernize this plugin and make it more aligned with the core's style & conventions?

**Comment by cmfcmf** [comment]
Thank you @duanwilliam for reporting this! I am currently blocked on missing type definitions for `@docusaurus/theme-translations`, see https://github.com/facebook/docusaurus/issues/6087.

@Josh-Cena I welcome contributions. Please feel free to open issues and/or pull requests. I have spent a lot of time last weekend to catch up with the latest Docusaurus development, but as you say, there is still stuff to do.

**Comment by cmfcmf** [comment]
Should be fixed in v0.9.0.
